src/core_plugins/testbed/public/testbed.js

```diff
@@ -2,11 +2,40 @@ import uiRoutes from 'ui/routes';
 import template from './testbed.html';
 import './testbed.less';
 
+import 'ui/sortable_column';
+
 uiRoutes.when('/testbed', {
   template: template,
   controllerAs: 'testbed',
   controller: class TestbedController {
-    constructor() {
+    constructor($scope, $injector) {
+
+      const $filter = $injector.get('$filter');
+      this.orderBy = $filter('orderBy');
+
+      this.items = [
+        { firstName: 'Jim', lastName: 'Unger' },
+        { firstName: 'CJ', lastName: 'Cenizal' },
+        { firstName: 'Shaunak', lastName: 'Kashyap' }
+      ];
+
+      // Initial sort state for the table
+      this.sortField = 'lastName';
+      this.sortReverse = false;
+
+      $scope.$watchMulti([
```

Comment:
Personally I've been trying to depend upon `$watchMulti` less, because the interface is so confusing (it's something we wrote, but it's presented as if it's part of Angular). In many cases, we can just use [$watchGroup](https://code.angularjs.org/1.4.14/docs/api/ng/type/$rootScope.Scope) instead. I know this is just a testbed, but I think we can do this in other cases where this kind of logic is used.